src/Agent.Worker/Container/ContainerInfo.cs

```diff
@@ -29,32 +33,85 @@ public ContainerInfo(IHostContext hostContext, Pipelines.ContainerResource conta
             this.ContainerCreateOptions = container.Properties.Get<string>("options");
             this.SkipContainerImagePull = container.Properties.Get<bool>("localimage");
             this.ContainerEnvironmentVariables = container.Environment;
```

Comment:
should this be `_environmentVariables=container.Environment;`?

Reply:
Doesnt matter to me, this was the existing code :)

Reply:
Made the change anyways, to support being able to clean up the private setter